src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/DotNetCoreProjectCompatibilityDetector.cs

```diff
@@ -228,6 +230,17 @@ private async Task WarnUserOfIncompatibleProjectAsync(CompatibilityLevel compatL
                     if (settingsManager != null)
                     {
                         suppressPrompt = settingsManager.GetValueOrDefault(SuppressDotNewCoreWarningKey, defaultValue: false);
+
+                        // Do not warn users about pre-release versions if they have checked "Use Preview SDKs"
+                        suppressPrompt |= settingsManager.GetValueOrDefault(UsePreviewSdkKey, defaultValue: false);
+                    }
+
+                    // If this is a pre-release version do not show the prompt for version compatibility
+                    var vsSetupConfig = new SetupConfiguration();
+                    ISetupInstance setupInstance = vsSetupConfig.GetInstanceForCurrentProcess();
+                    if (setupInstance is ISetupInstanceCatalog setupInstanceCatalog)
```

Comment:
Similarly, if the user is on 16.1 Preview, but we want them to update to 16.2 Preview, I think we still want this.

Instead, I think we want `IsPrerelease()` to be an input into the json file.